README.md

```diff
@@ -1,2 +1,20 @@
 # hw01
-Repository for homework 1
+
+![picture](PictureofMarie.jpg)
```

Comment:
It appears that the caption you attempted to add to this photo "picture" was not added in the file on GitHub. It does, however, appear when previewed in RStudio.